src/__stories__/list-story.tsx

```diff
@@ -64,9 +65,11 @@ type Args = {
     oneLineDescription: boolean;
     extraContent: boolean;
     disabled: boolean;
+    danger: boolean;
+    overInverse: boolean;
```

Comment:
the name for this control in other stories is "inverse" instead of "overInverse"

Reply:
I called it `overInverse` because otherwise it's difficult to understand the difference between `inverse` and `isInverse` (internal)